src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -146,6 +147,14 @@ to this directory.",
               .long("server-address")
               .required(false)
         )
+        .arg(
+          Arg::with_name("chunk-bytes")
+              .help("Number of bytes to include per-chunk when uploading bytes. grpc imposes a hard message-size limit of around 4MB.")
+              .takes_value(true)
+              .long("chunk-bytes")
+              .required(false)
+              .default_value(&format!("3 * 1024 * 1024"))
```

Comment:
This... confuses me a bit. Is `clap` going to parse this string and then evaluate it to get the default value?

Maybe it's correct, just pointing it out as odd.